src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -442,23 +442,20 @@ protected override async Task<bool> AcceptCommand(MessageId id, string method, J
                 case "Debugger.setPauseOnExceptions":
                     {
                         string state = args["state"].Value<string>();
-                        if (!context.IsRuntimeReady)
+                        switch (state)
                         {
-                            context.PauseOnCaught = false;
-                            context.PauseOnUncaught = false;
-                            switch (state)
-                            {
-                                case "all":
-                                    context.PauseOnCaught = true;
-                                    context.PauseOnUncaught = true;
-                                    break;
-                                case "uncaught":
-                                    context.PauseOnUncaught = true;
-                                    break;
-                            }
+                            case "all":
```

Comment:
Maybe use switch expression here?
```csharp
                        context.PauseOnExceptions = state switch
                        {
                            "all"      => PauseOnExceptionsKind.All,
                            "uncaught" => PauseOnExceptionsKind.Uncaught,
                            "none"     => PauseOnExceptionsKind.None,
                            _          => // error?
                        };
```

Reply:
done, and I set the default case to unset, I don't think we need to log an error. According to the documentation this should not happen: https://chromedevtools.github.io/devtools-protocol/tot/Debugger/#method-setPauseOnExceptions